README.md

```diff
@@ -37,6 +37,209 @@
 * Can work with Extensions
 * Can work with making a Class and Enums
 
+## JF NEW TEXT
+Just as in any object-oriented programming language, classes in Swift encapsulate data and functionality. In practical terms, that means properties that are variables (`var`) and constants (`let`) as well as methods. (Class methods are functions that are declared within a class. Functions can also be declared outside of classes.)
+
+>For more on classes insert Xref to Classes
+
+## Create a New Class - Person
+You can create a Person class.
+
+Encapsulated in that class will be three properties: name, height, and favoriteFood. Here are the property declarations:
+
+		var name: String
+		var height: Double
+		var favoriteFood: String
+
+You must provide some way of initializing those properties.  You could do that by declaring default values such as 
+
+		var name: String = ""
+
+It is more useful in many cases to provide a way for users to set initial values with an `init` method. Here’s what that looks like:
+
+	init(name: String, height: Double, favoriteFood: String) {
+        	self.name = name
+        	self.height = height
+        	self.favoriteFood = favoriteFood
+    	}
+
+Your Person class now looks like this.
+
+	class Person {
+
+    		// here are the properties
+    		var name: String
+    		var height: Double
+    		var favoriteFood: String
+
+    		// here is the initializer method
+    		init(name: String, height: Double, favoriteFood: String) {
+        		self.name = name
+        		self.height = height
+        		self.favoriteFood = favoriteFood
+    		}
+	}
+
+You can create an instance of this class and assign it to a local variable using this code;
+
+		let jim = Person(name: "Jim", height: 64.0, favoriteFood: "Pasta")
+
+## Create a Text Description Computed Variable for your Person Class
+How do you know that this works? How can you see what you’ve created? One way is to add a new property to the class.  Call it `textDescription` (because that’s what it will be). Here’s how you create that variable in your Class definition:
+
+		var textDescription: String {
+			return "Name: \(self.name) \nHeight: \(self.height) \nFavorite Food: \(self.favoriteFood)"
+		}
+ 
+## Create an Instance of the Person Class
+Now, you can create an instance of the class, and it will have the `textDescription` variable created automatically from the values you supply for the class. Here’s how you create an instance of the class, and then print `textDescription`.
+
+		let jim = Person(name: "Jim", height: 6.0, favoriteFood: “Pasta")
+		print (jim.description)
+
+Here’s the result
+
+		// Name: Jim Height: 6.0 Favorite Food: Pasta
```

Comment:
With how you defined the `textDescription` read-only computed property above, you included line breaks (\n) after each item.. so it would actually print like this:

```swift
// Name: Jim
// Height: 6.0
// Favorite Food: Pasta
```